### PR TITLE
Functions - Signum

### PR DESCRIPTION
**Pull request by Abduliitr**
## Checklist
- Solution is tested on sphere engine <br/>
    `Note:` Please copy only the solution part. Do not copy/paste the template code. That way, any issues with template code can be caught early on.
    - [x] Author
    - [x] Reviewer
- PR title is correct - [PR title conventions](https://github.com/DSA-solutions-the10xacademy/JavaScript-DSA-Solutions#title-for-a-pull-request)
    - [x] Author
    - [x] Reviewer
- Folder/File naming is correct - [Naming conventions](https://github.com/DSA-solutions-the10xacademy/JavaScript-DSA-Solutions#naming-of-files-and-folders)
    - [x] Author
    - [x] Reviewer
- Coding conventions are followed - [Coding conventions](https://github.com/DSA-solutions-the10xacademy/JavaScript-DSA-Solutions#coding-conventions)
    - [x] Author
    - [x] Reviewer
- Test cases (action item for **reviewer**) <br/>
    - To be ensured by checking the problem on sphere-engine
    - If any test cases are missing, please add them - [how to add new test cases](https://github.com/DSA-solutions-the10xacademy/JavaScript-DSA-Solutions#adding-new-test-cases)
    - [x] There are at least 2 test cases
    - [x] There is at least 1 test case that is different from the sample test cases

### CODE REVIEW

**Review by LVSsandeepVarma — APPROVED** [review]

File: solutions/functions/signum.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    }else{
+        result = 0;
+    }
+
+    return result;
+}
+
+// Please do not edit the code below this line.
+
+var test_input = parseFloat(readLine())
```
Comment (sailesh-ramayanam) [inline solutions/functions/signum.js:31]: @LVSsandeepVarma Please do not use `var`. Change the template to use `let`. Do this for all the widgets.

Comment (LVSsandeepVarma) [reply solutions/functions/signum.js:31]: replaced var  with  let

Comment (sailesh-ramayanam) [reply solutions/functions/signum.js:31]: @LVSsandeepVarma `semicolons` are missing at the end of line. Please do this for all the widgets and all lines.

Comment (LVSsandeepVarma) [reply solutions/functions/signum.js:31]: @sailesh-ramayanam  semicolons are  added

Comment (Abduliitr) [reply solutions/functions/signum.js]: Updated PR as per latest template.

**Review by sailesh-ramayanam — APPROVED** [review]

**Pull request merged**